shared/core/src/main/scala/org/hyperledger/identus/shared/utils/Json.scala

```diff
@@ -13,8 +21,29 @@ object Json {
     * @return
     *   canonicalized JSON string
     */
-
   def canonicalizeToJcs(jsonStr: String): Either[IOException, String] =
-    try { Right(new JsonCanonicalizer(jsonStr).getEncodedString) }
-    catch case exception: IOException => Left(exception)
+    try {
+      Right(new JsonCanonicalizer(jsonStr).getEncodedString)
+    } catch case exception: IOException => Left(exception)
+
+  /** Canonicalizes a JSON-LD string to RDF according to the Universal RDF Dataset Normalization Algorithm 2015
```

Comment:
@shotexa, why don't we use another canonization scheme for the JSON. For instance, https://datatracker.ietf.org/doc/html/rfc8785.html?